include/graylog_logger/LogUtil.hpp

```diff
@@ -28,14 +35,29 @@ enum class Severity : int {
   Debug = 7,
 };
 
+/// \brief Used to store multiple different types for the extra fields provided
+/// to the library.
 struct AdditionalField {
+  /// \brief Sets the instance of this struct to contain an empty string.
   AdditionalField() : FieldType(Type::typeStr), intVal(0), dblVal(0){};
+
+  /// \brief Sets the instance of this struct to contain a floating point value
+  /// (double).
+  /// \param[in] val The float value that will be stored by the struct.
```

Comment:
You could say _double_ or _floating-point_ instead of _float_, to avoid confusion.

Reply:
Done